frontend/src/metabase-types/api/dashboard.ts

```diff
@@ -1,5 +1,28 @@
+import {
+  ParameterTarget,
+  ParameterId,
+  Parameter,
+} from "metabase-types/types/Parameter";
+import { CardId, SavedCard } from "metabase-types/types/Card";
+
 export interface Dashboard {
   id: number;
   name: string;
   model?: string;
+  ordered_cards: DashboardOrderedCard[];
+  parameters: Parameter[];
 }
+
+export type DashboardOrderedCard = {
+  id: number;
+  card: SavedCard;
+  card_id: CardId;
+  parameter_mappings?: DashboardParameterMapping[];
+  series?: SavedCard[];
```

Comment:
These two properties show up everywhere, except the dashboard objects generated by x-rays. We should probably make a distinct type for those objects at some point.